src/interfaces.ts

```diff
@@ -45,4 +45,14 @@ export interface MnemonicOptions {
    * BIP44 index number
    */
   index?: number
+
+  /**
+   * Default path for HD wallet. 1023 by default
+   */
+  defaultPath?: string
+
+  /**
+   * Number of addresses
+   */
+  addresses?: number
```

Comment:
You can add a default value and change the name to `numberOfAddresses`:
```ts
...
numberOfAddresses = 1
...
```

Reply:
Got it!